docs/GUIdemo/tardis_example.yml

```diff
@@ -0,0 +1,57 @@
+# Example YAML configuration for TARDIS
```

Comment:
Can you make this a symbolic link that points to `docs/tardis_example.yml`?

Reply:
This pr is a demo for GUI and will not be merged. I'll keep things you mentioned in my mind when I make the guiding notebook. Thanks for your review!